source/ftrack_connect_pipeline_3dsmax/utils/max_alembic_commands.py

```diff
@@ -0,0 +1,48 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2022 ftrack
+
+from pymxs import runtime as rt
+
+# Alembic default options
+abc_default_import_options = {
+    "CoordinateSystem": 3,
+    "ImportToRoot": False,
+    "FitTimeRange": True,
+    "SetStartTime": False,
+    "UVs": True,
+    "Normals": True,
+    "VertexColors": True,
+    "ExtraChannels": True,
+    "Velocity": True,
+    "MaterialIDs": True,
+    "Visibility": True,
+    "CustomAttributes": True,
+    "ShapeSuffix": True,
+    "ObjectAttributes": True
+}
```

Comment:
This one is made because is too difficult to interact with python + max + abc so is for convenience on this particular integration. I wouldn't remove it because it benefits our self in order to work easier. But happy to add the same in other integrations.

Reply:
Lets leave it then, lets revisit this when why plugin:ify the utils later.